nowcasting_dataloader/datasets.py

```diff
@@ -367,6 +372,32 @@ def add_encodings(
                 x[key + "_query"] = future_encoding
         return x
 
+    def zero_out_nan_pv_systems(self, x: dict) -> dict:
```

Comment:
would you be able to put it as a method in BatchML. Then we wknow it will work for all datasets.py?

Reply:
Yeah, that might need to wait for moving the position encoding to BatchML too, as we want to zero out the encoding as well, which we can't do if its already been zero'd out before.

Reply:
I added an issue for it in #80